src/main/java/DataStorageInterface.java

```diff
@@ -0,0 +1,223 @@
+            }
+        }
+        return counter;
+    }
+
+    /**
+     * Returns Task Added Response with number of tasks Not Done.
+     *
+     * @param task Task to be added.
+     * @return Task Added Response with number of tasks Not Done.
+     */
+    public static String taskAdded(Task task) {
+        return "Got it. I've added this task:\n\t" + task.toString() +
+                String.format("\nNow you have %d tasks in the list",
+                        DataStorageInterface.getTasksNotDone());
+    }
+
+    /**
+     * Adds Todo Task to task list.
+     *
+     * @param query User entered title of Task.
+     * @return Task that is added.
+     * @throws WrongUsageException If todo command is used wrongly.
+     */
+    public static Task addToDo(String query) throws WrongUsageException {
+        Task newTask = new ToDo(query);
+        DataStorage.taskList.add(newTask);
+        return newTask;
+    }
+
+    /**
+     * Adds Deadline Task to task list.
+     *
+     * @param title User entered title of Task.
+     * @param preposition User entered preposition for date and time.
+     * @param date User entered due date of Task.
+     * @param time User entered timing of Task.
+     * @return Task that is added.
+     * @throws WrongUsageException If deadline command is used wrongly.
+     */
+    public static Task addDeadline(String title,
+                                   String preposition,
+                                   LocalDate date, LocalTime time) throws WrongUsageException {
+        Task newTask = new Deadline(title,preposition,date, time);
+        DataStorage.taskList.add(newTask);
+        return newTask;
+    }
+
+    /**
+     * Adds Event Task to task list.
+     *
+     * @param title User entered title of Task.
+     * @param preposition User entered preposition for date and time.
+     * @param date User entered due date of Task.
+     * @param time User entered timing of Task.
+     * @return Task that is added.
+     * @throws WrongUsageException If event command is used wrongly.
+     */
+    public static Task addEvent(String title, String preposition, LocalDate date, LocalTime time)
+            throws WrongUsageException {
+        Task newTask = new Event(title,preposition,date, time);
+        DataStorage.taskList.add(newTask);
+        return newTask;
+    }
+
+    /**
+     * Removes all tasks from the Task List.
+     */
+    public static void removeAllTasks() {
+        DataStorage.taskList.clear();
+    }
+
+    /**
+     * Adds Tasks from an array to the current Task List.
+     *
+     * @param taskArrayList Tasks to be Added in current Task List.
+     */
+    public static void loadNewTasks(ArrayList<Task> taskArrayList) {
+        DataStorage.taskList.addAll(taskArrayList);
+    }
+
+    /**
+     * Returns String representation of all tasks in the task list.
+     *
+     * @return String representation of all tasks in the task list.
+     */
+    public static String listOfTasks() {
```

Comment:
Consider adding a verb to your method name? Maybe `printListOfTasks()`? The coding standard states "Names representing methods must be verbs" and that would be more intuitive for the reader as well.